.github/workflows/tests-integration.yml

```diff
@@ -75,4 +75,4 @@ jobs:
     - name: Tests
       if:  ${{ github.event_name != 'schedule' && github.event_name != 'workflow_dispatch' }}
       run: |
-        pytest tests -m "integration"
+        pytest tests -m "integration and not slow"
```

Comment:
Should we run "slow" tests on scheduled CI? Otherwise, we may not be able to detect when these slow tests fail.

Reply:
Yes, we run `slow` and `not slow` tests when we specify `-m "integration"`.